conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,118 @@
+to address the situation. In ongoing situations, any member may decide to employ
+any of the tools available to the committee, including bans and blocks.
+
+If the incident involves physical danger, any member of the committee may -- and
+should -- act unilaterally to protect the safety of those involved. This can
+include contacting law enforcement (or other local personnel) and speaking on
+behalf of the Jupyter Steering Council.
+
+In situations where an individual committee member acts unilaterally, they must
+report their actions to the committee for review within 24 hours.
+
+
+## Resolutions
+
+The committee must agree on a resolution by consensus. If the group cannot reach
+consensus and deadlocks for over a week, the group will turn the matter over to
+the Steering Council for resolution.
+
+Possible responses may include:
+
+* Taking no further action (if we determine no violation occurred).
+* A private reprimand from the committee to the individual(s) involved. In this
+  case, the group chair will deliver that reprimand to the individual(s) over
+  email, cc'ing the group.
+* A public reprimand. In this case, the committee chair will deliver that
+  reprimand in the same venue that the violation occurred (i.e. in the gitter
+  chat room for a gitter chat room violation; email for an email violation,
+  etc.). The group may choose to publish this message elsewhere for
+  documentation purposes.
+* An imposed vacation (i.e. asking someone to "take a week off" from a mailing
+  list or gitter chat room). The committee chair will communicate this
+  "vacation" to the individual(s). They'll be asked to take this vacation
+  voluntarily, but if they don't agree then a temporary ban may be imposed to
+  enforce this vacation.
+* A permanent or temporary ban from some or all Jupyter spaces (mailing lists,
+  gitter.im, etc.). The group will maintain records of all such bans so that
+  they may be reviewed in the future or otherwise maintained.
+* A request for a public or private apology. The chair will deliver this
+  request. The committee may, if it chooses, attach "strings" to this request:
+  for example, the group may ask a violator to apologize in order to retain
+  one’s membership on a mailing list.
+
+Once a resolution is agreed upon, but before it is enacted, the committee will
+contact the original reporter and any other affected parties and explain the
+proposed resolution. The committee will ask if this resolution is acceptable,
+and must note feedback for the record. However, the committee is not required to
+act on this feedback.
+
+Finally, the committee will make a report to the Jupyter Steering Council (as
+well as the Jupyter core team in the event of an ongoing resolution, such as a
+ban).
+
+The committee will never publicly discuss the issue; all public statements
+(aside from the responses listed above if those are on public channels) will be
+made by the Jupyter Steering Council.
```

Comment:
I agree. "...the chair of the Code of Conduct Committee or the Jupyter Steering Council".